packages/toolpad-core/package.json

```diff
@@ -33,8 +33,8 @@
     "url": "https://github.com/mui/mui-toolpad/issues"
   },
   "dependencies": {
-    "@mui/x-data-grid-pro": "^5.15.2",
-    "@tanstack/react-query": "^4.1.0",
+    "@mui/x-data-grid-pro": "^5.17.1",
```

Comment:
@Janpot i think this broke the datagrids

Reply:
we should add some integration test to see if datagrid work sometime soon

Reply:
In what sense? They seemed to work for me. Can you open a ticket?

Reply:
i've created an issue for the bug, hopefully it's not just some local issue i'm having but didn't seem like it
bug: https://github.com/mui/mui-toolpad/issues/921
integration test issue: https://github.com/mui/mui-toolpad/issues/920